vcap/vcap/testing/capsule_loading.py

```diff
@@ -23,6 +25,13 @@ def mock_init(self, filter_func):
 
     with mock.patch('vcap.device_mapping.DeviceMapper.__init__',
                     mock_init):
-        capsule: BaseCapsule = load_capsule(path=packaged_capsule_path)
+        if from_memory:
```

Comment:
What's the benefit off having the from_memory, if you're going to load it from the filesystem anyways?